setup.py

```diff
@@ -6,14 +6,13 @@
 
 requirements = [
     "deprecated",
-    "gpflow>=2.6.3",
-    "numpy",
+    "gpflow>=2.9.2",
+    "numpy<2",
```

Comment:
What is the issue with `numpy>=2`?

Reply:
It's a major, breaking release: e.g. see https://numpy.org/devdocs/numpy_2_0_migration_guide.html.

We'll certainly want to have a conversation about supporting it at some point (possibly company wide), but at this point it probably makes sense to just wait until the dust settles (it was release just a few days ago).